Add make-release.sh script to automate the release process

mkuznyetsov opened the pull request:
Signed-off-by: Mykhailo Kuznietsov <mkuznets@redhat.com>

### What does this PR do?
Add make-release.sh script to automate certain steps of release process

### What issues does this PR fix or reference?

mkuznyetsov committed:
Add make-release.sh script to automate the release process

Signed-off-by: Mykhailo Kuznietsov <mkuznets@redhat.com>

nickboldt commented on make-release.sh:
```diff
@@ -0,0 +1,113 @@
+popd > /dev/null || exit
+
+# cleanup tmp dir
+# cd /tmp && rm -fr "$TMP"
```
are you leaving this commented until you've had a chance to run this and know it works ? :)

mkuznyetsov replied:
woops :smile:

nickboldt commented on make-release.sh:
```diff
@@ -0,0 +1,113 @@
+  git fetch origin "${BRANCH}:${BRANCH}"
+  git checkout "${BRANCH}"
+fi
+
```
when do you replace 7.9.0-SNAPSHOT with 7.9.0 in the 7.9.x branch?

mkuznyetsov replied:
it is done by centos ci job, so this operation is not needed here

nickboldt replied:
please add a comment and pointer to the centos ci job/script so it's more clear and in 6mo when we've both forgotten all about this it'll be easier to remember it :)

mkuznyetsov committed:
fixup! Add make-release.sh script to automate the release process

nickboldt committed:
Update make-release.sh

update comments re: when -snapshot suffix is removed, as it's not done explicitly here (unlike other make-release.sh scripts)

nickboldt committed:
Update RELEASE.md

nickboldt reviewed (approved):
+1 after fixes to comments/doc
